src/components/MultiHopTrade/components/SharedTradeInput/SharedTradeInput.tsx

```diff
@@ -10,7 +10,7 @@ import { useSharedHeight } from '../TradeInput/hooks/useSharedHeight'
 type SharedTradeInputProps = {
   bodyContent: JSX.Element
   footerContent: JSX.Element
-  hasUserEnteredAmount: boolean
```

Comment:
This is only ever used to determine if the `sideComponent` should open, so re-named accordingly so it's saner to re-use.